org.zdoom.GZDoom.yaml

```
@@ -0,0 +1,108 @@
+app-id: org.zdoom.GZDoom
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: "3.32"
+command: gzdoom.sh
+rename-icon: gzdoom
+
+finish-args:
+- --device=dri
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=x11
+- --share=ipc
+- --socket=pulseaudio
+
+# Where would wads be?
+- --filesystem=xdg-desktop:ro
+- --filesystem=xdg-documents:ro
+- --filesystem=xdg-download:ro
+
+# Screenshots? 
+# Sceenshots go by default in ~/.config/
+#- --filesystem=xdg-pictures
+
+# Based on GZDoom
+# We access the original ~/.config/gzdoom because
+# that is where users should put their iwads
+- --env=DOOMWADDIR=/app/share/games/doom
+- --filesystem=~/.config/gzdoom
+
+cleanup:
+- /app/include
+- /app/lib/*.a
+- /app/lib/*.la
+- /app/lib/pkgconfig
+
+modules:
+
+- name: ncurses
+  no-autogen: true
+  config-opts:
+  - --prefix=/app
+  make-install-args:
+  - install.libs
+  cleanup:
+  - /bin
+  - /include
+  - /share/man
+  - "/lib/*.a"
+  - "/lib/*.la"
+  sources:
+  - type: archive
+    url: http://ftp.gnu.org/gnu/ncurses/ncurses-6.1.tar.gz
+    sha256: aa057eeeb4a14d470101eff4597d5833dcef5965331be3528c08d99cebaa0d17
+
+- name: fluidsynth
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+  - -DLIB_SUFFIX=
+  # https://lists.nongnu.org/archive/html/fluid-dev/2010-10/msg00057.html
+  - -DCMAKE_EXE_LINKER_FLAGS=-lncurses
+  - -DCMAKE_SHARED_LINKER_FLAGS=-lncurses
+  sources:
+  - type: archive
+    url: https://github.com/FluidSynth/fluidsynth/archive/v1.1.11.tar.gz
+    sha256: da8878ff374d12392eecf87e96bad8711b8e76a154c25a571dd8614d1af80de8
+
+# This builds both the application code and
+# the custom game data
+- name: gzdoom
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+  - -DSEND_ANON_STATS=OFF
+  # Required when targeting 32-bit x86
+  - -DCMAKE_CXX_FLAGS="-msse3"
+  sources:
+  - type: archive
+    url: https://github.com/coelckers/gzdoom/archive/g4.2.0.tar.gz
+    sha256: 9d781e868f32ad53aa262691f76cab09b12661ababd2b7f5735fe852ab74d566
+  - type: file
+    url: https://github.com/coelckers/gzdoom/raw/g4.2.0/soundfont/gzdoom.sf2
+    sha256: fca3e514b635a21789d4224e84865d2954a2a914d46b64aa8219ddb565c44869
+  - type: shell
+    commands:
+    - install -Dm 644 gzdoom.sf2 /app/share/sounds/sf2/gzdoom.sf2
+
+- name: launcher
+  buildsystem: simple
+  sources:
+  - type: script
+    commands:
+    - gzdoom +fluid_patchset /app/share/sounds/sf2/gzdoom.sf2 -file lights.pk3 brightmaps.pk3 $@
+    dest-filename: gzdoom.sh
+  - type: file
+    path: org.zdoom.GZDoom.desktop
+  - type: file
+    path: org.zdoom.GZDoom.appdata.xml
+  - type: file
+    path: gzdoom.png
+  build-commands:
+  - install -D gzdoom.sh /app/bin/gzdoom.sh
+  - install -Dm 644 org.zdoom.GZDoom.desktop -t /app/share/applications
+  - install -Dm 644 org.zdoom.GZDoom.appdata.xml -t /app/share/appdata
```

```suggestion
  - install -Dm 644 org.zdoom.GZDoom.appdata.xml -t /app/share/metainfo
```
the `$datadir/appdata` shouldn't be used anymore.